contracts/test/BasicOwnable.sol

```diff
@@ -0,0 +1,8 @@
+pragma solidity ^0.5.8;
```

Comment:
Does this need to exist at all? `Ownable` should be itself not-abstract.

Reply:
I thought I tried to deploy an `Ownable` instance and couldn't, but looking at it now I can't see how that would be the case. We should be able to remove `BasicOwnable`